dockers/actions-plan-preview/main.go

```diff
@@ -71,20 +97,20 @@ func main() {
 	}
 	log.Println("Successfully retrieved plan-preview result")
 
-	body := makeCommentBody(event, result)
-	comment, err := sendComment(
-		ctx,
-		ghClient,
-		event.Owner,
-		event.Repo,
-		event.PRNumber,
-		body,
-	)
-	if err != nil {
-		log.Fatal(err)
+	// Maybe the PR is already closed so Piped could not clone the source code.
+	if result.HasError() {
+		pr, err := getPullRequest(ctx, ghClient, event.Owner, event.Repo, event.PRNumber)
+		if err != nil {
+			log.Fatal(err)
+		}
+		if !pr.GetClosedAt().IsZero() {
+			doComment(failureBadgeURL + "It was unable to run plan-preview for a closed pull request.")
```

Comment:
```suggestion
			doComment(failureBadgeURL + "Unable to run plan-preview for a closed pull request.")
```
unify with the others 🙏

Reply:
👍